tests/oss-fuzz/build.sh

```diff
@@ -41,27 +41,23 @@ set -eu
 # Build dav1d with sanitizer flags.
 # Adds extra flags: -Db_sanitize=$SANITIZER -Db_lundef=false, and -Denable_asm=false for msan
 DAV1D_EXTRA_FLAGS=""
-if [ "$SANITIZER" != "coverage" ] && [ "$SANITIZER" != "introspector" ]
-then
+if [[ "$SANITIZER" != "coverage" && "$SANITIZER" != "introspector" ]]; then
```

Comment:
This is recommended in the Google Shell Style Guide:
https://google.github.io/styleguide/shellguide.html#s5.4-loops